### PR TITLE
UTs: Remove OldVerifier.VerifyNonConcurrentAnalyzer

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
This is draft for now. Rebase will be needed after the merge of #5294 (current target branch of this PR) and #5293 (see comment below)

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs [hunk 1]
```diff
@@ -147,7 +147,7 @@ public static class OldVerifier
                 .WithConcurrentAnalysis(false)
                 .WithOptions(ParseOptionsHelper.CSharpPreview)
                 .Verify();
-
+        //FIXME: Remove after rebase
```
Comment (pavel-mikula-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs:150]: This overload will be removed after #5293 gets merged and this PR rebased

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/SerializationConstructorsShouldBeSecuredTest.cs [hunk 2]
```diff
@@ -30,23 +30,23 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
```
Comment (csaba-sagi-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Rules/SerializationConstructorsShouldBeSecuredTest.cs:30]: NITPICK: Two unused usings.

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldNotBeIgnoredTest.cs [hunk 3]
```diff
@@ -29,68 +29,57 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
```
Comment (csaba-sagi-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldNotBeIgnoredTest.cs:29]: One unused using here too.

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/XmlExternalEntityShouldNotBeParsedTest.cs [hunk 4]
```diff
@@ -18,10 +18,11 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
```
Comment (csaba-sagi-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Rules/XmlExternalEntityShouldNotBeParsedTest.cs:18]: Aaaaand 4 here :)

File: analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs (hunk 1 above)
Comment (csaba-sagi-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/OldVerifier.cs:150]: #5293 is merged, so you can rebase and fix the part that is begging with a FIXME :)

**Review by csaba-sagi-sonarsource — CHANGES_REQUESTED** [review]
Left a few comments, please take a look.

**Comment by csaba-sagi-sonarsource** [comment]
@pavel-mikula-sonarsource  Am I missing something, it seems to me that the unused usings were not removed?

**Comment by pavel-mikula-sonarsource** [comment]
@csaba-sagi-sonarsource Indeed, I commit the review update to my local backup branch instead 🤦‍♂️

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**